### DISCUSSION TITLE
Accessing SLC data

### DISCUSSION THREAD

**Discussion by Spiruel** [ESA-PhiLab/OpenSarToolkit#93] [Q&A] [answered]
Can this package be used to programmatically acquire SLC time series data? 

I am looking to downloading multiple image patches, distributed globally and spanning a calendar year. Is this something that is supported here? 

Platforms such as GEE, SentinelHub etc only cater to GRD products.

Thanks

**Comment by BuddyVolly** [comment, marked as answer]
Yes, use the ASF mirror for that. Need to register at https://search.asf.alaska.edu/
And check the notebooks( particularly Tutorial 2) here: https://opensartoolkit.readthedocs.io/en/latest/example/data_search_%26_access.html

**Reply by Spiruel** [reply]
Thanks!
